components/trade-activities/TradeActivitiesTable.tsx

```diff
@@ -175,7 +177,6 @@ export function TradeActivitiesTable({
         {mergedActivities?.length ? (
           mergedActivities.map((activity, index) => (
             <ActivityRow
-              key={getActivityId(activity)}
```

Comment:
Why did you delete this?

Reply:
not used in ActivityRow (not even declared as a parameter). i used as 'id' for testing, but I removed it for the commit, because it wasn't in the HTML in the first place